builder/amazon/common/access_config.go

```diff
@@ -86,3 +100,24 @@ func (c *AccessConfig) Prepare(t *packer.ConfigTemplate) []error {
 
 	return nil
 }
+
+func GetInstanceMetaData(path string) (contents []byte, err error) {
+	url := "http://169.254.169.254/latest/meta-data/" + path
```

Comment:
Hey @jen20 we are probably going to triage Packer in a few days. Can you keep an eye on the upstream and, if 182 is merged, update here accordingly? If not, we will merge as is and can update in the future. How does that sound?